src/components/gauge/readme.md

```diff
@@ -0,0 +1,37 @@
+# gx-gauge
+
+This component allows you to display information in the form of ranges, and decide whether you want to show it using linear or circular gauges.
```

Comment:
```suggestion
This component allows displaying information in the form of ranges, and decide whether you want to show it using linear or circular gauges.
```

Reply:
Ready! Fix it in commit  fd99fb8dc921c4e781ec9f22a6daac7000004d47